src/viur_cli/local.py

```diff
@@ -46,6 +47,13 @@ def run(profile, additional_args):
                    f"Please install the 'gcloud' tool or Log in with an appropriate account.")
 
     conf = config.get_profile(profile)
+    additional_args = list(additional_args)
+
+    if conf["port"]:
+        additional_args.append(f"--port={conf['port']}")
+    if conf["gunicorn_port"]:
```

Comment:
If they are optional an missing this would crash:
```suggestion
    if conf.get("port"):
        additional_args.append(f"--port={conf['port']}")
    if conf.get("gunicorn_port"):
```